google/cloud/spanner/doc/spanner-main.dox

```diff
@@ -25,9 +25,9 @@ into your project.
 - [Client::Read()](@ref google::cloud::spanner::Client::Read()) - to read the
   rows in a table.
 - @ref spanner-error-handling to learn how the library reports run-time errors.
-- @ref spanner-env-logging enable logging. THis can be an effective approach to
-  diagnose many problems.
 - @ref spanner-env for environment variables affecting the library.
+- @ref spanner-env includes environment variables to enable logging. This can be
+  an effective approach to diagnose many problems.
 - @ref spanner-endpoint-example
 - @ref spanner-auth-example
```

Comment:
This leads to a page where two links are not separated by any space.

Reply:
Fixed.  I used the same structure as the generated documents, if you want we can make it nicer as this is a hand-crafted library.